### ISSUE TITLE
Empty closed ports list

### ISSUE THREAD

**Issue by LennyLip** [shpak86/tcp_scanner#2]
Why I get the closed ports list empty? the host is reachable as you can see.

```
I/flutter (19317): HTTP ports scan result
I/flutter (19317): Host:          192.168.4.1
I/flutter (19317): Scanned ports: [503, 513]
I/flutter (19317): Open ports:    [503]
I/flutter (19317): Closed ports:  []
I/flutter (19317): Elapsed time:  0.783s
```

**Comment by shpak86** [comment]
Hey @LennyLip!
A port can be skipped if it is filtered or remote host closes the connection abnormally. I'll fix this soon.

**Comment by shpak86** [comment]
Fixed in version 1.1.2.
